pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -297,6 +298,77 @@ func (r *ReconcileStorageCluster) newStorageClasses(initData *ocsv1.StorageClust
 	return ret, nil
 }
 
+// newRBDSnapshotClass returns a SnapshotClass for RBD backing
```

Comment:
looks like the comment is not complete

Reply:
Done, kindly check whether my wordings are correct... (=